ts/themes/SessionTheme.tsx

```diff
@@ -18,3 +20,26 @@ export const SessionTheme = ({ children }: { children: any }) => (
     {children}
   </>
 );
+
+export async function ensureThemeConsistency(): Promise<boolean> {
+  const theme = window.Events.getThemeSetting();
+
+  return new Promise(resolve => {
+    ipcRenderer.send('get-native-theme');
+    ipcRenderer.once('send-native-theme', (_, shouldUseDarkColors) => {
+      const isMismatchedTheme = isThemeMismatched(theme, shouldUseDarkColors);
+      if (isMismatchedTheme) {
+        const newTheme = getOppositeTheme(theme);
+        void switchThemeTo({
+          theme: newTheme,
+          mainWindow: true,
+          usePrimaryColor: true,
+          dispatch: window?.inboxStore?.dispatch || undefined,
```

Comment:
`window?.inboxStore?.dispatch` already resolves to undefined I believe, no need for the undefined?

Reply:
fixed